schemas/user.json

```diff
@@ -117,7 +117,10 @@
 		},
 		"givenName": {
 			"type": "string",
-			"multilingual": true
+			"multilingual": true,
+			"validation": [
+				"nullable"
+			]
```

Comment:
I am wondering what is the difference between now and earlier (without nullable) :thinking:

Reply:
I've tested this using the context UI (it uses the API under the hood, so perhaps, different validations than different parts of the system) and a multilingual field. This is the behavior that I got:
- Required without nullable: All locales must be filled.
- Required with nullable: Only the main locale is required.

Basically, all fields in the system which are multilingual, also have this nullable validation, so I've just added to the ones that were missing.

Reply:
Yes... I also understood it so from the code... but could not test it with user form... Maybe because it is working differently...
:+1: